core-service/src/main/java/org/fundacionjala/sevenwonders/beans/GameService.java

```diff
@@ -98,4 +96,25 @@ public Game getGame(int id){
         return games.get(id);
     }
 
+    public Player getPlayer(int id, PlayerModel playerModel){
+        return games.get(id).getPlayers().stream().filter(b ->b.getName().equals(playerModel.getUserName())).findAny()									// If 'findAny' then return found
+                .orElse(null);
+    }
+
+    public void addChooseCard(ChooseCardModel chooseCardModel) {
+        Game game = getGame(chooseCardModel.getId());
+        List<Player> currentList = new ArrayList();
```

Comment:
You don't need to initialize the list if you will assign a new value in the next line